WooCommerce/Classes/Extensions/Array+Helpers.swift

```diff
@@ -13,6 +13,11 @@ extension Array {
 
         return removeFirst()
     }
+
+    /// A Boolean value indicating whether an array has characters.
```

Comment:
Since this is not the method for checking if an array has characters, maybe it can be: `A Boolean value indicating whether the collection is not empty.` 👍

Reply:
Thanks for reviewing it, Paolo. 
I will change this comment line as yours make more sense. Also, I took I while to decide whether I should or not apply those changes for the optionals. 

As soon as we have a decision about it I can change and commit it again.

Reply:
@pmusolino Paolo, I accidentally committed the file to fix the issue #1618 on this pull request